pkg/operator/status_test.go

```diff
@@ -102,3 +104,84 @@ func TestOperatorStatusProgressing(t *testing.T) {
 		assert.True(t, condition.LastTransitionTime.Equal(&conditionAfterAnotherSync.LastTransitionTime), "test-case %v expected LastTransitionTime not to be updated if condition state is same", i)
 	}
 }
+
+func TestGetOrCreateClusterOperator(t *testing.T) {
+	var namespace = "openshift-machine-api"
```

Comment:
nit: I would chose something else; something else because we genuinely use this value and we want to assert we can plug in any value.